### PR TITLE
Custom loader support

### PR DESCRIPTION
**Pull request by Holzhaus**
Fixes https://github.com/sivakov512/python-markdown-full-yaml-metadata/issues/18

### CODE REVIEW

**Comment by Holzhaus** [comment]
@sivakov512 firendly ping

**Comment by sivakov512** [comment]
Oh, sorry for long (very long) answer.
Do you think that in your case custom loader is better then boolean config option like `full_loader`?

**Comment by sivakov512** [comment]
I see that CI does not work here, so code can not be tested. I will fix CI in 1-2 weeks.

**Comment by sivakov512** [comment]
@Holzhaus 
I set up a new CI and refactored the project organization a little. Please add tests for this feature.

> Do you think that in your case custom loader is better then boolean config option like full_loader?
What do you think?

**Comment by Holzhaus** [comment]
For my use case, the boolean flag suffices, other people might want to pass a customized subclass. The boolean flag would be less flexible, but it's your decision ;-)

**Comment by sivakov512** [comment]
Okay, your solution sounds better than flag. But it should be tested and documented a bit in `readme`.

**Comment by Holzhaus** [comment]
Requesting a review is disabled, so pinging you here.

File: tests/test_loaders.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            {
+                "title": "What is Lorem Ipsum?",
+                "category": "Lorem Ipsum",
+                "date": datetime.datetime(2020, 1, 1, 10, 0, 0),
+                "num_comments": 5,
+            },
+        ],
+    ),
+)
+def test_default_loader(
```
Comment (sivakov512) [inline tests/test_loaders.py:58]: Is is not about default loader, maybe `test_custom_loader`?

File: tests/test_loaders.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    md = markdown.Markdown(
+        extensions=["full_yaml_metadata"],
+        extension_configs={
+            "full_yaml_metadata": {
+                "yaml_loader": loader,
+            }
+        },
+    )
+
+    assert md.convert(SOURCE) == EXPECTED_BODY
```
Comment (sivakov512) [inline tests/test_loaders.py:71]: This test should check only loader overriding, so remove this assert pls.

**Review by sivakov512 — CHANGES_REQUESTED** [review]

**Comment by Holzhaus** [comment]
I'll take care of #21 after this has been merged.

**Review by sivakov512 — APPROVED** [review]

**Comment by sivakov512** [comment]
@Holzhaus, I had an error in CI due to which nothing starts here. Upgrade to the master, please.

**Comment by sivakov512** [comment]
@Holzhaus thanks a lot!

**Pull request merged**